site/next.config.mjs

```diff
@@ -34,6 +43,10 @@ module.exports = {
           },
         ],
       },
+      {
+        source: "/:path*",
```

Comment:
Does this override the above headers that I set specifically for /api/block-rate then?

Reply:
Nope, we're all good here, Header declarations have an additive effect. The function accepts `Headers[]` to allow more configuration granularity between pages.